### DISCUSSION TITLE
Profitability viewer

### DISCUSSION THREAD

**Discussion by hanseartic** [kalepail/stellar-quest-bounties#89] [Bounty Ideas]
# Secenario
When an account is used for SDEX trading it is desirable to see at a glance the profit/loss ratio of that account.

# Bounty idea
There could be a bounty to visualize the profitability of a trading account.
At first this may seem simple – just query horizon for account's trades. The pitfalls may wait down the road where the correct pairs need to be matched.
Overall here the graphical presentation also plays an important role. Therefore this could also be a team-bounty.

Happy to discuss/sharpen this draft of a potential bounty.

**Comment by ralphilius** [comment]
Wow, I had this same thought as well. Good to see you bring this up 🥇

**Reply by hanseartic** [reply]
Thank you. As you've already thought about it, do you already have some ideas we should not miss in such a bounty?

**Comment by kalepail** [comment]
Seems cool! Keep fleshing it out. Maybe some rough sketch or example to help guide the thought process for this discussion?
